kolibri/plugins/utils/urls.py

```diff
@@ -10,13 +10,16 @@ def get_urls():
     for plugin_instance in registered_plugins:
         url_module = plugin_instance.url_module()
         api_url_module = plugin_instance.api_url_module()
+        root_url_module = plugin_instance.root_url_module()
         instance_patterns = []
         # Normalize slug
         slug = plugin_instance.url_slug().lstrip("^").rstrip("/") + "/"
         if url_module:
             instance_patterns += i18n_patterns(url_module.urlpatterns, prefix=slug)
         if api_url_module:
             instance_patterns.append(url(slug + "api/", include(api_url_module)))
+        if root_url_module:
+            instance_patterns.append(url("", include(root_url_module)))
```

Comment:
What I had suggested was to not mount these within the plugin namespace (so they are doubly 'root' - both in terms of their point of mounting, and the Django namespace in which they are mounted).

Would want to extract this specific part into a separate 'root_urls' function that gets separately called and mounted in the `deployment.default.urls` configuration. This should allow us to remove a lot of the monkey patching above.